tests/lodash-extras-tests.js

```diff
@@ -0,0 +1,256 @@
+import { expect } from 'chai';
+import sinon from 'sinon';
+import _ from 'lodash';
+import { all } from '../../../dist/server/index';
+
+describe(
+  'Lodash-extras tests',
+  function () {
+    beforeEach( () => {
+      _.runInContext().merge(_, all);
+    });
+
+    describe('isPresent', () => {
+      it('exists', () => expect(!_.isUndefined(_.isPresent) && !_.isNull(_.isPresent)).to.be.true);
+
+      it('Identifies present objects', () => {
+        const testObject = { };
+        expect(_.isPresent(testObject)).to.be.true;
+      });
+
+      it('Identifies undefined objects', () => {
+        let testObject;
+        expect(_.isPresent(testObject)).to.be.false
+      });
+
+      it('Identifies null objects', () => {
+        const testObject = null;
+        expect(_.isPresent(testObject)).to.be.false
+      });
+    });
+
+    describe('isBlank', () => {
+      it('exists', () => expect(_.isPresent(_.isBlank)).to.be.true);
```

Comment:
I would say we should check all primitive types here as well- but knowing how it's implemented, it isn't critical as verifying `isPresent`